### PR TITLE
LA-371 Apply tablet design for workgroup

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/116990855-e5f9f880-acfd-11eb-962e-d40aafcddb04.mov

### CODE REVIEW

File: lib/presentation/util/helper/responsive_widget.dart [hunk 1]
```diff
@@ -36,8 +36,10 @@ class ResponsiveWidget extends StatelessWidget {
   final Widget mediumScreen;
   final Widget smallScreen;
 
-  static final int minLargeWidth = 960;
-  static final int minMediumWidth = 640;
+  static final int minLargeWidth = 950;
+  static final int minMediumWidth = 600;
```
Comment (hoangdat) [inline lib/presentation/util/helper/responsive_widget.dart:40]: why we need to change this?

File: lib/presentation/util/helper/responsive_widget.dart [hunk 2]
```diff
@@ -36,8 +36,10 @@ class ResponsiveWidget extends StatelessWidget {
...
+
+  static final double horizontalPaddingLargeWidth = 132.0;
```
Comment (hoangdat) [inline lib/presentation/util/helper/responsive_widget.dart:42]: can you explain to me?

File: lib/presentation/view/order_by/order_by_button.dart [hunk 3]
```diff
@@ -60,16 +61,18 @@ class OrderByButtonBuilder {
...
-          _currentSorter.orderType.getSortIconPath(imagePath),
-          width: 15,
-          height: 15,
-          fit: BoxFit.fill,
-          color: AppColor.primaryColor,
-        ),
-      ),
+      contentPadding: ResponsiveWidget.isLargeScreen(_context)
+          ? EdgeInsets.symmetric(horizontal: 155)
+          : EdgeInsets.symmetric(horizontal: 16) ,
```
Comment (hoangdat) [inline lib/presentation/view/order_by/order_by_button.dart:66]: srr, I can not understand

File: lib/presentation/util/helper/responsive_widget.dart (hunk 2 above)
Comment (dab246) [reply lib/presentation/util/helper/responsive_widget.dart:42]: For a large tablet screen by design, we need to indent the listview items in the middle. So I set the value to all widgets that I need to use.

File: lib/presentation/util/helper/responsive_widget.dart (hunk 1 above)
Comment (dab246) [reply lib/presentation/util/helper/responsive_widget.dart:40]: These are the two most common values for large and xlarge tablet screens. You can see more here. [https://developer.android.com/training/multiscreen/screensizes](https://developer.android.com/training/multiscreen/screensizes)

File: lib/presentation/view/order_by/order_by_button.dart (hunk 3 above)
Comment (dab246) [reply lib/presentation/view/order_by/order_by_button.dart:66]: We need a distance from the left margin on the large tablet screen

File: lib/presentation/util/helper/responsive_widget.dart (hunk 1 above)
Comment (hoangdat) [reply lib/presentation/util/helper/responsive_widget.dart:40]: what is the unit for this width? (pixel? ...)

Comment (hoangdat) [reply lib/presentation/util/helper/responsive_widget.dart:40]: and IMO 950 is not a common value

Comment (dab246) [reply lib/presentation/util/helper/responsive_widget.dart:40]: it is dp

Comment (dab246) [reply lib/presentation/util/helper/responsive_widget.dart:40]: I have read through the documentation it is a popular desktop value. I don't remember the link of that document. Maybe we will discuss this value again.

Comment (hoangdat) [reply lib/presentation/util/helper/responsive_widget.dart:40]: iOS using dp too?

Comment (dab246) [reply lib/presentation/util/helper/responsive_widget.dart:40]: In flutter we use the same px. If dp is large then dp ~ px

**Pull request merged**